.pre-commit-config.yaml

```diff
@@ -14,6 +14,7 @@ repos:
           - id: clean-notebooks
             name: clean-notebooks
             files: \.ipynb$
+            exclude: ^tutorials/evals/.*\.ipynb$
```

Comment:
@axiomofjoy if it's simpler, we should probably just do this via pre-commit based CI rather than hatch